/**:
  ros__parameters:
    time_buffer: 1.5  # [s] required time without collision in forward simulation
    distance_buffer: 1.5  # [m] distance from obstacles to consider a collision
    min_adjusted_velocity: 1.5  # [m/s] minimum velocity that the module can set
    max_deceleration: 2.0  # [m/sÂ²] maximum deceleration caused by the adjusted velocity
    start_distance: 0.0  # [m] distance ahead of ego from which to start modifying the trajectory
    downsample_factor: 10  # factor by which to downsample the input trajectory for calculation
    obstacle_type: occupancy_grid  # type used for obstacle detection. Must be 'pointcloud' or 'occupancy_grid'
    occupancy_grid_obstacle_threshold: 60 # occupancy grid values higher than this are considered to be obstacles
    dynamic_obstacles_buffer: 1.0 # [m] extra distance around dynamic obstacles used to mask the occupancy grid and avoid false obstacle detection
    dynamic_obstacles_min_vel: 0.5 # [m/s] velocity above which a dynamic obstacle is ignored by the module

    forward_projection:
      model: bicycle  # model to use for forward projection at each trajectory point. Either "particle" or "bicycle"
      # parameters used only with the bicycle model
      steering_offsets: [-0.01, 0.0, 0.01]  # [rad] steering angle offsets used to model uncertainty in the forward projection
      nb_points: 5  # number of points representing the curved projections
